contracts/src/accounts/SimpleAccountFactory_7560.sol

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.23;
+
+import "@openzeppelin/contracts/utils/Create2.sol";
+import "@openzeppelin/contracts/proxy/ERC1967/ERC1967Proxy.sol";
+
+import "./SimpleAccount_7560.sol";
+
+/**
+ * A sample factory contract for SimpleAccount
+ * A UserOperations "initCode" holds the address of the factory, and a method call (to createAccount, in this sample factory).
+ * The factory's createAccount returns the target account address even if it is already installed.
+ * This way, the entryPoint.getSenderAddress() can be called either before or after the account is created.
+ */
+contract SimpleAccountFactory_7560 {
+    SimpleAccount_7560 public immutable accountImplementation;
+
+    constructor() {
+        accountImplementation = new SimpleAccount_7560();
+    }
+
+    /**
+     * create an account, and return its address.
+     * returns the address even if the account is already deployed.
+     * Note that during UserOperation execution, this method is called only if the account is not deployed.
+     * This method returns an existing account address so that entryPoint.getSenderAddress() would work even after account creation
+     */
+    function createAccount(
+        address owner,
+        uint256 salt
+    ) public returns (SimpleAccount_7560 ret) {
+        address addr = getAddress(owner, salt);
+        uint256 codeSize = addr.code.length;
+        if (codeSize > 0) {
+            return SimpleAccount_7560(payable(addr));
+        }
+        ret = SimpleAccount_7560(
+            payable(
+                new ERC1967Proxy{salt: bytes32(salt)}(
+                    address(accountImplementation),
+                    abi.encodeCall(SimpleAccount_7560.initialize, (owner))
+                )
+            )
+        );
+    }
+
+    /**
+     * calculate the counterfactual address of this account as it would be returned by createAccount()
+     */
+    function getAddress(
+        address owner,
+        uint256 salt
+    ) public view returns (address) {
+        return
+            Create2.computeAddress(
+                bytes32(salt),
+                keccak256(
+                    abi.encodePacked(
+                        type(ERC1967Proxy).creationCode,
+                        abi.encode(
+                            address(accountImplementation),
+                            abi.encodeCall(
+                                SimpleAccount_7560.initialize,
+                                (owner)
+                            )
+                        )
```

Comment:
Wouldn't it be enough to include all the codeCall for the initialize, but only the owner of the factor?
```suggestion
                        abi.encode(
                            address(accountImplementation),
                            owner
                        )
```